src/wrappers/themis/jsthemis/package.json

```diff
@@ -1,11 +1,17 @@
 {
   "name": "jsthemis",
   "version": "0.12.0",
-  "description": "Themis is a data security library, providing users with high-quality security services for secure messaging of any kinds and flexible data storage. Themis is aimed at modern developers, with high level OOP wrappers for Ruby, Python, PHP, Java / Android and iOS / OSX. It is designed with ease of use in mind, high security and cross-platform availability.",
+  "description": "Themis is a convenient cryptographic library for data protection.",
```

Comment:
Do we have any restrictions on description length? 

I thought the goal of updating description in packages was to unify them, and this text is very different from the one we used in `build.gradle` and texts updated by previous PR #587.

Let's make all descriptions consistent

Reply:
Yes, as we've found out with @ilammy, there are
a) noticeable restrictions regarding the overall description size for packages on npmjs.com and the full existing description would not look good and will be cut off abruptly
b) esthetic restrictions regarding the description size - everything that is present in the "description" field in the json file will be crammed into a pop-up next to the package name in the search field on npmjs.com
The solution for this is to have these 2 really breve descriptions for JsThemis and for WasmThemis with the standardised description put into the readme files, featured prominently by npmjs.com.

Reply:
got it, makes sense, thank you!